index.html

```diff
@@ -1013,21 +1165,31 @@ <h2>Terms of Use</h2>
 
       </section>
 
-
       <section>
         <h2>Evidence</h2>
-        <p>Evidence information for the credential in the
-        Verifiable Credentials Model may be provided by adding the
-        following <a>property</a>:
+
+        <p>
+The <em>evidence</em> property is used by an <a>issuer</a> to describe the
+set of evidence that was used when determining whether or not to issue
```

Comment:
"when determining" or "to determine"?

Reply:
Fixed in dc3b3b19cc8ad888d81bedcb932130ffe38825d6.